packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -49,8 +58,364 @@ describe('validateOnlyAudits', () => {
           { slug: 'b', title: 'B' },
           { slug: 'c', title: 'C' },
         ],
-        'd',
+        'missing-audit',
+      ),
+    ).toThrow(new AuditsNotImplementedError(['missing-audit']));
+  });
+});
+
+describe('validateOnlyCategories', () => {
+  it('should not throw for category slugs existing in given categories', () => {
+    expect(
+      validateOnlyCategories(
+        [
+          {
+            slug: 'g1',
+            title: 'G 1',
+            refs: [
+              { slug: 'a', weight: 1 },
+              { slug: 'b', weight: 1 },
+              { slug: 'c', weight: 1 },
+            ],
+          },
+          {
+            slug: 'g2',
+            title: 'G 2',
+            refs: [
+              { slug: 'd', weight: 1 },
+              { slug: 'e', weight: 1 },
+              { slug: 'f', weight: 1 },
+            ],
+          },
+        ],
+        'g2',
+      ),
+    ).toBeTruthy();
+  });
```

Comment:
Two things here that apply to both tests of `validateOnlyCategories`.
1. No need to add so many group references as the function does not care about them so it unnecessarily bloats the test.
2. Please use realistic values, example provided for generic categories (to be replaced by real life scenario). It is much more readable that way.

```suggestion
    expect(
      validateOnlyCategories(
        [
          {
            slug: 'performance',
            title: 'Performance',
            refs: [{ slug: 'speed-index', weight: 1 }],
          },
          {
            slug: 'coverage',
            title: 'Code coverage',
            refs: [{ slug: 'function-coverage', weight: 1 }],
          },
        ],
        'performance',
      ),
    ).toBeTruthy();
  });
```

Reply:
done updating ALL test data :)